doc/src/build/move/build-test.md

```diff
@@ -0,0 +1,344 @@
+---
+title: Build and Test the Sui Move Package
+---
+
+## Building a package
+
+Ensure you are in the `my_move_package` directory that contains your package, and then use the following command to build it:
+
+``` shell
+$ sui move build
+```
+
+A successful build returns a response similar to the following:
+
+```shell
+Build Successful
+Artifacts path: "./build"
+```
+
+Now that we have designed our asset and its accessor functions, let us
+test the code we have written.

```

Comment:
> Sorry if this is due to being new, but where does the design of the asset and its accessor functions happen? This is about building a package, no code was written yet (in this topic at least). I see mention of this in Write a package. Should this line be moved to that file instead? Or is building and testing the package the way to test the code? If in 2 different topics perhaps we need to link to the other topic where this info is included rather than treat it as a step earlier in the doc?

As with the comment above, this is outside the scope of this change. Delaying it to answer every possible question will no doubt result in merge conflicts and cascading delays. This PR merely moves this text.

Let's stick to reviewing that action here and track your desired clarifications in a new issue owned by you and resolved in PRs authored by you. I am absolutely certain our engineering team will help. Thanks!